matrix-sdk-android/src/main/java/org/matrix/android/sdk/internal/session/sync/handler/SyncResponsePostTreatmentAggregatorHandler.kt

```diff
@@ -82,7 +83,9 @@ internal class SyncResponsePostTreatmentAggregatorHandler @Inject constructor(
             }
         }
         if (hasUpdate) {
-            updateUserAccountDataTask.execute(UpdateUserAccountDataTask.DirectChatParams(directMessages = directChats))
+            tryOrNull("Unable to update user account data") {
```

Comment:
If it fails will it retry next sync or is lost?

Reply:
If it fails, it will not be retried on next sync.